include/cvc5/cvc5_proof_rule.h

```diff
@@ -2498,6 +2498,32 @@ enum ENUM(ProofRewriteRule)
    * \endverbatim
    */
   EVALUE(ARRAYS_SELECT_CONST),
+
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Arrays -- Macro distinct arrays**
+   *
+   * .. math::
+   *   (A=B) = \bot
+   *
+   * where :math:`A` and :math:`B` are distinct array values.
```

Comment:
I think these descriptions are too vague...

Reply:
Macro rules shouldn't require a precise explaination, but I can add that the internals are such that `Node::isConst` returns true for both?